src/components/TopBar/ColorBoard.tsx

```diff
@@ -1,17 +1,20 @@
-import React from "react";
+import React, { useEffect } from "react";
```

Comment:
I didn't find useEffect in your code. Is it needed?

Reply:
thanks i have deleted it